src/PublishPort.jsx

```diff
@@ -47,7 +88,11 @@ export const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount })
                             min={1}
                             max={65535}
                             value={item.hostPort || ''}
+                            validated={validationFailed?.hostPort ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230713-111536-64696d17-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)